src/sass/components/_donation-form.scss

```
@@ -0,0 +1,56 @@
+input,
+textarea {
+    font-family: inherit;
+    font-size: inherit;
+}
+
+input[type="checkbox"]:checked {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    border: 6px solid $green;
+}
+
+input[type="text"]:not(:placeholder-shown):invalid,
+input[type="tel"]:not(:placeholder-shown):invalid {
+    border-color: red;
+}
+
+label,
+.donate-now-button {
+    font-family: inherit;
+    font-size: inherit;
+    display: block;
+    margin-top: 2px;
+}
+
+.donate-now-button {
+    background-color: $green;
+    border: 8px solid $green;
+    color: $yellow;
+    font-weight: 600;
+    margin: 20px auto 0 auto;
+
+    :hover {
+        font-weight: 600;
+    }
```

```suggestion
    &:hover {
        font-weight: 600;
    }
```

You need to use the ampersand here to get CSS that looks like this: `.donate-now-button:hover`. The way it's currently written, the CSS selector you create is this: `.donate-now-button :hover` and that space is important -- it means any element _inside_ the `.donate-now-button` class that is hovered; without the space, it means when the `.donate-now-button` element is hovered.